doc/tutorials/3d/point_cloud/point_cloud.markdown

```diff
@@ -0,0 +1,75 @@
+Point cloud visualisation {#tutorial_point_cloud}
+==============================
+
+|    |    |
+| -: | :- |
+| Original author | Klepikov Dmitrii |
+| Compatibility | OpenCV >= 5.0 |
+
+Goal
+----
+
+In this tutorial you will:
+
+-   Load and save point cloud data
+-   Visualise your data
+
+Requirements
+------------
+
+For visualisations u need to complile OpenCV library with OpenGL support. For this you should set WITH_OPENGL flag on in cmake
+
+Practice
+-------
+
+Loading and saving of point cloud can be done using loadPointCloud(savePointCloud)
+
+Currently supported formats are:
+- .OBJ (supported keys are v(which is responcible for point position), vn(normal coordinates) and f(faces of a mesh), other keys are ignored)
+- .PLY (all encoding types(ascii and byte) are supported with limitaion to only float type for data)
+
+@code{.py}
+vertices, normals = cv2.loadPointCloud("teapot.obj")
+@endcode
+
+Function loadPointCloud returns vector of points of float (cv::Point3f) and vector of their normals(if specified in source file)
```

Comment:
Please use \` \` for function and type names. Doxygen ads links for them to API reference.